### PR TITLE
Add questions for arithmetic case study

### PR DESCRIPTION
**Pull request by yilinwei**

### CODE REVIEW

File: case-study/language/exercises/1-arithmetic.md [hunk 1]
```diff
@@ -22,8 +22,27 @@ Our very first interpreter will parse numbers and evaluate them to ... the numbe
...
 At the moment you don't need to worry about precedence.
+
+## Questions
+
+
+1. What is a program? What is an interpreter?
+
+
+
+2. Without parsing, what would the expression `1 + 2 * 56 + 8` be represented as as an `Expr`?
```
Comment (noelwelsh) [inline case-study/language/exercises/1-arithmetic.md]: I don't understand what this question is asking. Perhaps you can reword it?

File: case-study/language/exercises/1-arithmetic.md [hunk 2]
```diff
@@ -22,8 +22,27 @@ Our very first interpreter will parse numbers and evaluate them to ... the numbe
...
+   Is it what you would expect?
+
+
+
+3. Discuss the similarities between the signature and structure of `prettyPrint` and `eval`.
```
Comment (noelwelsh) [inline case-study/language/exercises/1-arithmetic.md]: `prettyPrint` is in the solution, they aren't asked to write their own in the current exercise, so they should be directed to look at the solution before answering this question.

Writing `prettyPrint` might make a good exercise and naturally leads into property-based testing IMO.

File: case-study/language/exercises/1-arithmetic.md (hunk 1 above)
Comment (yilinwei) [reply case-study/language/exercises/1-arithmetic.md]: Reworded it; I'm trying to get them to think about how the parser actually represents the expression and how that will affect the `eval` recursion.

File: case-study/language/exercises/1-arithmetic.md (hunk 2 above)
Comment (yilinwei) [reply case-study/language/exercises/1-arithmetic.md]: :+1:  changed it.

**Pull request merged**